flux-tests/tests/lib/rvec.rs

```diff
@@ -116,13 +117,15 @@ impl<T> Iterator for RVecIter<T> {
 impl<T> std::ops::Index<usize> for RVec<T> {
     type Output = T;
 
+    #[flux::assume]
```

Comment:
Why did these have to become `assume`?

Reply:
There are many tests that import this file so this code is checked multiple times. This makes it very annoying if you ever break this code because many tests will spuriously fail.